client/state/site-address-change/actions.js

```diff
@@ -146,7 +145,7 @@ export const requestSiteAddressChange =
 				}
 			);
 
-			const newSlug = get( data, 'new_slug' );
+			const newSlug = data?.new_slug;
```

Comment:
Looking at the `site-address-change` endpoint, `data` should always be there and a valid object, but I guess it won't hurt if we bet on the safer side.

Reply:
I thought the same thing, just in case 🤞🏻